extensions/interactions/ItemSelectionInput/webdriverio.js

```diff
@@ -76,8 +76,8 @@ var submitAnswer = async function(elem, answer) {
 
   for (var i = 0; i < answerArray.length; i++) {
     var desiredAnswer = answerArray[i];
-    var selectionInputItem = elem.$(
-      `.e2e-test-item-selection-input-item=${desiredAnswer}`)
+    var selectionInputItem = await elem.$(
+      `.e2e-test-item-selection-input-item*=${desiredAnswer}`)
```

Comment:
`e2e-test-item-selection-input-item` have some other text as well along with desiredAnswer so we need to use partialText checker. Previously this was not throwing any error because this code is used at only one place (in extension test suite) which is migrated in this PR.